### PR TITLE
chore: update to node 16

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
-  Upgraded to Node 16

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

DD-{number} #done

### CODE REVIEW

**Comment by KyleTryon** [comment]
Hey folks, i tried this out locally and noticed that I am getting the NPM install error there as well.

image


Looks like we are looking for `@dailydotdev/eslint-config` externally though I see it is bundled in as a lerna package.

**Comment by rebelchris** [comment]
@KyleTryon I forgot to put it back in one commit, should be re-added now.
The main change of this PR is switching from Node 14 to 16, it seems to create some peer dependency issue, but for some reason I can install everything locally.

**Comment by KyleTryon** [comment]
> @KyleTryon I forgot to put it back in one commit, should be re-added now. The main change of this PR is switching from Node 14 to 16, it seems to create some peer dependency issue, but for some reason I can install everything locally.

I have tested again with the latest code in your branch, and I am not able to install it locally. Have you fully deleted your node_modules folder locally before re-installing?

image

```
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@dailydotdev%2feslint-plugin-daily-dev-eslint-rules - Not found
npm ERR! 404 
npm ERR! 404  '@dailydotdev/eslint-plugin-daily-dev-eslint-rules@*' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
```

I have found several examples of similar issues reported with lerna but none that I have seen yet offer much for solutions.

Do we know, is this package meant to be in the registry? Is lerna supposed to be using the local packages rather than attempting to fetch them remotely?

**Comment by rebelchris** [comment]
Before releasing it's crucial we do extended tests locally for all three compartments.
webapp - extension and companion.

Also sidenote to update the docs and install steps.

**Review by idoshamun — APPROVED** [review]

**Comment by idoshamun** [comment]
Gitpod perhaps as well

**Comment by rebelchris** [comment]
@sshanzel @kirillkurko would love to also get your review on this one

File: packages/shared/src/components/sidebar/Sidebar.spec.tsx [hunk 1]
```diff
@@ -135,32 +127,30 @@ it('should not render create my feed button if user has alerts.filter as false',
...
-
-  await waitFor(() => {
-    const key = getFeedSettingsQueryKey(defaultUser);
-    const data = client.getQueryData(key) as AllTagCategoriesData;
-    expect(getHasAnyFilter(data.feedSettings)).toBeTruthy();
-  });
-
-  expect(updateAlerts).toBeCalledWith({ filter: false });
-  expect(mutationCalled).toBeTruthy();
+  // await act(async () => {
```
Comment (sshanzel) [inline packages/shared/src/components/sidebar/Sidebar.spec.tsx:141]: Just wanted to check if this was expected to be commented out?

**Review by sshanzel — APPROVED** [review]
This is amazing! Thank you for the effort in pushing to v16!!! 🚢 

Just had one question regarding a commented out test case.

File: packages/shared/src/components/sidebar/Sidebar.spec.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/sidebar/Sidebar.spec.tsx:141]: Oh good point, I think this was actually meant to be refactored as `act` is not optimal.
Let me double-check this one 🤯

**Review by kirillkurko — APPROVED** [review]

**Pull request merged**